src/main/kotlin/org/bundleproject/utils/Assets.kt

```diff
@@ -0,0 +1,18 @@
+package org.bundleproject.utils
+
+import io.ktor.client.request.*
+import org.bundleproject.json.assets.ModAssets
+import org.bundleproject.json.assets.ModSource
+import org.bundleproject.json.assets.ModVersion
+
+suspend fun fetchAssets(): ModAssets = httpClient.get(assetsUrl)
+suspend fun resolveUrl(urlData: ModVersion): String {
+    return when (urlData.source) {
+        ModSource.DIRECT -> urlData.ref
+        ModSource.GITHUB -> {
```

Comment:
?

Reply:
This will resolve a ModVersion data class to a static download url (not done yet, I just wanted to push my current code so you could give some feedback)